common-rendering/src/components/MetaData.tsx

```diff
@@ -0,0 +1,363 @@
+	const tokens = bylineAsTokens(byline, tags);
+
+	const bylineComponents = getBylineComponentsFromTokens(tokens, tags);
+
+	const renderedTokens = bylineComponents.map((bylineComponent) => {
+		if (typeof bylineComponent === "string") {
+			return bylineComponent;
+		}
+		return (
+			<ContributorLink
+				contributor={bylineComponent.token}
+				contributorTagId={bylineComponent.tag.id}
+				key={bylineComponent.tag.id}
+			/>
+		);
+	});
+
+	return <>{renderedTokens}</>;
+}; */
+
+/*
+	Foreach author in authors
+	if string contains name replace with authorlink then return <p> byline with links
+*/
+
+/* export const getContributorTags = (tags: TagType[]): TagType[] => {
+	return tags.filter((t) => t.type === "Contributor");
+}; */
+
+/* const getAuthorName = (tags: TagType[]) => {
+	const contributorTag = tags.find((tag) => tag.type === "Contributor");
+	return contributorTag && contributorTag.title;
+}; *
+
+type AuthorLinkDetails = {
+	id: string;
+	name: string;
+};
+
+const getAuthorNames = (tags: TagType[]) => {
+	const names: AuthorLinkDetails[] = [];
+	tags.find((tag) => {
+		if (tag.type === "Contributor") {
+			names.push({ name: tag.title, id: tag.id });
+		}
+	});
+};
+
+/*function Byline(byline: string, authors: Authors[]) {
+	if (byline.includes("now")) {
+		byline.replace("now", "(now)");
+	}
+	if (byline.includes("earlier")) {
+		byline.replace("earlier", "(earlier)");
+	}
+}*/
+
+const ContributorLink: React.FC<{
+	contributor: string;
+	contributorTagId: string;
+}> = ({ contributor, contributorTagId }) => (
+	<a
+		rel="author"
+		data-link-name="auto tag link"
+		href={`//www.theguardian.com/${contributorTagId}`}
+	>
+		{contributor}
+	</a>
+);
+
+// Shamelessly borrowed from Lucy! (This should be lifted out as it's very useful!)
+function getThemeColours(theme: Theme) {
+	switch (theme) {
+		case Pillar.Sport:
+			return {
+				background: sport[100],
+				color: sport[100],
+			};
+		case Pillar.Culture:
+			return {
+				background: culture[200],
+				color: culture[200],
+			};
+		case Pillar.Lifestyle:
+			return {
+				background: lifestyle[200],
+				color: lifestyle[200],
+			};
+		case Pillar.Opinion:
+			return {
+				background: opinion[200],
+				color: opinion[200],
+			};
+		default:
+			return {
+				background: news[200],
+				color: news[200],
+			};
+	}
+}
+
+const bylineStyle = (colour) => css`
+	color: ${text.ctaPrimary};
+	font-style: italic;
+	${headline.xxsmall()}
+	line-height: 115%;
+
+	a {
+		font-weight: 700;
+		${from.desktop} {
+			color: ${colour};
+		}
+		${until.desktop} {
+			color: ${text.ctaPrimary};
+		}
+		text-decoration: none;
+		font-style: normal;
+		:hover {
+			text-decoration: underline;
+			background: #ffe500;
+			color: #121212;
+		}
+	}
+`;
+
+const dateStyle = (colour) => css`
+	color: ${colour};
+	${textSans.xsmall()}
+	:hover {
+		cursor: pointer;
+	}
+
+	${from.desktop} {
+		color: ${palette.neutral[20]};
+	}
+`;
+
+// I would normally use the Lines component
```

Comment:
This comment doesn't really explain _why_  you've chosen to not use the `lines` component - an explanation of that would make a more useful comment.